emmet-core/emmet/core/vasp/calculation.py

```diff
@@ -618,7 +622,7 @@ def from_vasp_files(
         strip_bandstructure_projections: bool = False,
         strip_dos_projections: bool = False,
         store_volumetric_data: Optional[Tuple[str]] = None,
-        store_trajectory: bool = False,
+        store_trajectory: Union[bool, str] = False,
```

Comment:
Instead of mixing `bool` and `str` inputs to support a third "partial" option, I would vote change this to be an enum value or string representation of an enum value. That way the type and options are more well-defined.

Reply:
I switched to the Enum. Passing the string value is also fine. Let me know if you have other suggestions or better naming conventions.

Reply:
Looks fine to me, let me know when you are happy to have it merged.

Reply:
If it is fine, you can proceed to merge. Thanks!